internal/handlers/common/msg_whatsmyuri.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"context"
+
+	"github.com/FerretDB/FerretDB/internal/clientconn/info"
```

Comment:
@AlekSi this is not according to the issue description, but it looked quite strange to import the whole `clientconn` to handlers, so I came up with a totally separate package. What do you think?

Reply:
Sounds good

Reply:
But let's rename it to `clientconninfo` or something, because it will be used as `info.Something`, and that's too generic